src/dev/python/NativeCall.py

```diff
@@ -3,11 +3,11 @@
 from ctypes import *
 import os
 # for windows
-#dll_core = CDLL(os.getcwd() + '\dsn.core.dll')
-#dll_helper = CDLL(os.getcwd() + '\dsn.dev.python_helper.dll')
+dll_core = CDLL(os.getcwd() + '\dsn.core.dll')
```

Comment:
You can simply commit, and you will see the commit in this pull request so you don't need to file another one.